vpgl/algo/pyvpgl_algo.cxx

```diff
@@ -33,13 +38,34 @@ vgl_point_3d<double> wrap_bproj_plane(CAM_T const& cam,
   return output;
 }
 
+// NOTE pyvxl_holder_types is implicitly converting the vil_image_resource_sptr
+// to a vil_image_resource when the object is passed between python and C++
+// See vil/pyvil.cxx:crop_image_resource
+// Alternatively, take both a vil_image_view and a vpgl_geo_camera
+vgl_point_3d<double> wrap_bproj_dem(vpgl_rational_camera<double> const& rcam,
```

Comment:
It looks like `vpgl_backproject::bproj_plane` is templated on camera type in VXL - could you do `template<class CAM_T>` (same as `wrap_bproj_plane`)?

Reply:
Ya. I pushed a change.

But to create bindings for other camera types, besides vpgl_rational_camera, (which would take advantage of the `bproj_dem` overload that accepts a vpgl_camera) we'd have to make a change to vxl to create another overload of `bproj_dem` that accepts a const reference to a vpgl_camera like the one in vpgl_backproject for `bproj_plane`

```cpp
static bool bproj_dem(vpgl_camera<double> const& cam,
```